src/anomalib/utils/visualization/image.py

```diff
@@ -324,6 +359,16 @@ def generate(self) -> np.ndarray:
             axis.imshow(image_dict["image"], image_dict["color_map"], vmin=0, vmax=255)
             if image_dict["title"] is not None:
                 axis.title.set_text(image_dict["title"])
+            if image_dict["descr"] is not None:
+                wrapped_text = textwrap.fill(
+                    image_dict["descr"],
+                    width=70 // num_cols,
+                )  # Adjust 'width' based on your subplot size and preference
+
+                axis.set_title(wrapped_text, fontsize=10)
```

Comment:
Would this override the value provided in line 316?

Reply:
My fault, I had to merge both into one, since is the title of the plot I leave title=description.